cpp/tests/strings/find_tests.cpp

```diff
@@ -198,6 +196,37 @@ TEST_F(StringsFindTest, ContainsLongStrings)
   CUDF_TEST_EXPECT_COLUMNS_EQUIVALENT(*results, expected);
 }
 
+TEST_F(StringsFindTest, MultiContains)
+{
+  using cudf::test::iterators::null_at;
+  auto const strings = cudf::test::strings_column_wrapper{
+    {"Héllo, there world and goodbye",
+     "quick brown fox jumped over the lazy brown dog; the fat cats jump in place without moving",
+     "the following code snippet demonstrates how to use search for values in an ordered range",
+     "it returns the last position where value could be inserted without violating the ordering",
+     "algorithms execution is parallelized as determined by an execution policy. t",
+     "he this is a continuation of previous row to make sure string boundaries are honored",
+     "abcdefghijklmnopqrstuvwxyz 0123456789 ABCDEFGHIJKLMNOPQRSTUVWXYZ !@#$%^&*()~",
+     "",
+     ""},
+    null_at(8)};
+  auto strings_view = cudf::strings_column_view(strings);
+  std::vector<std::string> match_targets({" the ", "a", ""});
```

Comment:
Maybe throw in some targets with multi-byte characters. Especially at the beginning. Same for the benchmarks too perhaps. Example of a multi-byte character is `é`

Reply:
Added a `é` target for both UT and benchmark test, add passed.